#! /bin/sh /usr/share/dpatch/dpatch-run
## 99-unnamed.dpatch by  <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: mysql_upgrade should only read [mysql_upgrade] not [client].

@DPATCH@

# From: holyfoot at mysql dot com
# Date: July 3 2006 11:23am
# Subject: bk commit into 5.0 tree (holyfoot:1.2212) BUG#19452
# 
# Below is the list of changes that have just been committed into a local
# 5.0 repository of hf. When hf does a push these changes will
# be propagated to the main repository and, within 24 hours after the
# push, to the public repository.
# For information on how to access the public repository
# see http://dev.mysql.com/doc/mysql/en/installing-source-tree.html
# 
# ChangeSet
#   1.2212 06/07/03 14:22:39 holyfoot@deer.(none) +1 -0
#   bug #19452 (mysql_upgrade reads [client] section but doesn't handle [host] option
# 
#   client/mysql_upgrade.c
#     1.4 06/07/03 14:15:14 holyfoot@stripped +1 -1
#     "client" group removed from the 'readlist'
# 
# This is a BitKeeper patch.  What follows are the unified diffs for the
# set of deltas contained in the patch.  The rest of the patch, the part
# that BitKeeper cares about, is below these diffs.
# User:	holyfoot
# Host:	deer.(none)
# Root:	/home/hf/work/mysql-5.0.19452

--- 1.3/client/mysql_upgrade.c	Tue May  2 08:59:44 2006
+++ 1.4/client/mysql_upgrade.c	Mon Jul  3 14:15:14 2006
@@ -65,7 +65,7 @@
 };
 static const char *load_default_groups[]=
 {
-  "mysql_upgrade", "client", 0
+  "mysql_upgrade", 0
 };
 
 #include <help_end.h>
